test/com/linkedin/drelephant/tuning/AutoTunerApiTestRunner.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.drelephant.tuning;
+import static org.junit.Assert.*;
+import static play.test.Helpers.*;
+import static common.DBTestUtil.*;
+import static common.DBTestUtil.*;
+
+public class AutoTunerApiTestRunner implements Runnable {
```

Comment:
Suggestion: Do you plan to use this class across tests? If not, just add it as a private static class inside TuningManagerTest. If you intend to use it elsewhere in a later PR, then fine